ckanext/recombinant/templates/recombinant/snippets/choices_reference.html

```diff
@@ -1,4 +1,4 @@
-<h3>{{ _("Field choices reference") }}</h3>
```

Comment:
If there is a reason for it we can translate the "Field choices reference" message and "(required)" separately, we already have a translation for the latter.